roles/download/defaults/main.yml

```diff
@@ -125,12 +125,17 @@ crictl_checksums:
 # Checksums
 kubelet_checksums:
   arm:
+    v1.18.9: 24f7559fe5214b5f3b625ab035e29159441e6cfd248befbeb78b63e660fccd23
+    v1.18.8: 831f50ea45384884c50395c288d493e75dd134a962dc95261ce122de5e6a17ec
+    v1.18.6: 5f8367f9f5de77c022ec5d0cd86e897b7a33318185eaadb7736033d8dabcdbdb
```

Comment:
just a question: How to maintain versions on this list of version hashes?
This question comes because v1.18.7 is skipped.

/cc @oomichi

Reply:
1.18.7 is a NO-OP version as 1.18.7 was a buggy tag (as was 1.17.10) so we don't have hashes for it as it shouldn't be use at all (pointing to a beta version IIRC)

And to maintain it you have 2 scripts in `script` folder to get hashes from binary (even if it's only a `shasum -a 256`)